packages/react-styles/package.json

```diff
@@ -17,6 +17,8 @@
     "css": "^2.2.3",
     "cssom": "^0.3.4",
     "cssstyle": "^0.3.1",
+    "emotion": "^9.2.6",
```

Comment:
Shouldn't we remove aphrodite from `package.json` so we don't have two packages for one thing?

Reply:
Yep we should good catch.